vue-frontend/src/components/portfolio/FooterSection.vue

```diff
@@ -12,9 +12,9 @@
       />
     </aspect-ratio>
   </section>
-
+  <section><UserReviewSection v-bind:json="response.review" /></section>
```

Comment:
This should be in portfolio's main page, check how other components are added

Reply:
i have seen other components but we have to that footer is one component and we have to add it in between it that's why added like this